beacon-chain/core/state/transition.go

```diff
@@ -427,6 +475,68 @@ func ProcessBlockNoVerifyAttSigs(
 	return state, nil
 }
 
+// ProcessBlockNoVerify creates a new, modified beacon state by applying block operation
+// transformations as defined in the Ethereum Serenity specification. It does not validate
+// any block signature except for deposit and slashing signatures. It also returns the relevant
+// signature set from all the respective methods.
+//
+// Spec pseudocode definition:
+//
+//  def process_block(state: BeaconState, block: BeaconBlock) -> None:
+//    process_block_header(state, block)
+//    process_randao(state, block.body)
+//    process_eth1_data(state, block.body)
+//    process_operations(state, block.body)
+func ProcessBlockNoVerify(
```

Comment:
Why does this need to be exported?

Reply:
Probably an annoying requirement of testing, since tests don't live in the same package. This is also the case for 
`ProcessBlockNoVerifyAttSigs`